sdk/formrecognizer/ai-form-recognizer/src/transforms.ts

```diff
@@ -53,22 +58,33 @@ export function toTextLine(original: TextLineModel, pageNumber: number): FormLin
         text: w.text,
         boundingBox: toBoundingBox(w.boundingBox),
         confidence: w.confidence || 1,
-        pageNumber: pageNumber
+        pageNumber
       };
     })
   };
 
   return line;
 }
 
+export function toSelectionMark(original: SelectionMark, pageNumber: number): FormSelectionMark {
+  return {
+    kind: "selectionMark",
+    pageNumber,
+    boundingBox: toBoundingBox(original.boundingBox),
+    confidence: original.confidence,
+    state: original.state
+  };
+}
+
 export function toFormPage(original: ReadResultModel): FormPage {
   return {
     pageNumber: original.pageNumber,
     textAngle: original.angle,
     width: original.width,
     height: original.height,
     unit: original.unit,
-    lines: original.lines?.map((l) => toTextLine(l, original.pageNumber))
+    lines: original.lines?.map((l) => toTextLine(l, original.pageNumber)),
+    selectionMarks: original.selectionMarks?.map((m) => toSelectionMark(m, original.pageNumber))
```

Comment:
Should `tables` be added here? Or is the transform somewhere else?

Reply:
It's a little convoluted but `tables` are inserted into the pages in the `toFormPages` transform that's one level up, since we need more than the ReadResultModel to compute their values.